### ISSUE TITLE
Form doesn't load on webpack after moment update

### ISSUE THREAD

**Issue by josefernand** [formio/formio.js#212]
The last release `v2.19.2` includes a moment update `(2.19.0)` https://github.com/formio/formio.js/commit/bb49bbbc0bbcec22f668afdac1ca2f68938659cc that contains a bug for webpack:
Please check: https://github.com/moment/moment/issues/4216
The form doesn't load and the console only shows a warning message:
```
./node_modules/moment/src/lib/locale/locales.js
Module not found: Can't resolve './locale' in ....
```
I just confirmed doing a downgrading the moment version to:
```
"moment": "~2.18.1",
```

**Comment by travist** [comment]
I will push up a fix tonight.

**Comment by travist** [comment]
Fixed in 2.19.3. Thanks.
